cluster/gce/gci/configure-helper.sh

```diff
@@ -484,17 +484,17 @@ function ensure-local-ssds-ephemeral-storage() {
   safe-format-and-mount "${device}" "${ephemeral_mountpoint}"
 
   # mount container runtime root dir on SSD
-  local container_runtime="${CONTAINER_RUNTIME:-containerd}"
-  systemctl stop "$container_runtime"
+  local container_runtime_name="${CONTAINER_RUNTIME_NAME:-containerd}"
```

Comment:
these jobs using `KUBE_CONTAINER_RUNTIME=containerd` which eventually gets assigned to `CONTAINER_RUNTIME`
https://cs.k8s.io/?q=CONTAINER_RUNTIME%3Dcontainerd&i=nope&files=&excludeFiles=&repos=
should they be migrated to use KUBE_CONTAINER_RUNTIME_NAME?

as `systemctl stop remote` won't work

Reply:
I don't think so, because the default is `containerd`.
The whole `KUBE_CONTAINER_RUNTIME=containerd` seems to be an error that was never fixed. https://github.com/kubernetes/test-infra/pull/24918 will remove any mention of `CONTAINER_RUNTIME`, `KUBE_CONTAINER_RUNTIME` and `KUBE_WINDOWS_CONTAINER_RUNTIME` from test scenarios.
Maybe there would be some extra tweaking needed for CRI-O tests.
Some extra work on setting the defaults based on the value of `KUBE_CONTAINER_RUNTIME_ENDPOINT` may also be nice in the future.